.github/workflows/ci_win.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         include:
           - name: "Visual Studio 16 2019"
             os: windows-2019
-            toolset: '14.28'
+            toolset: '14.29'
```

Comment:
Supposedly, you are able to just say `14.xx` to use whatever is available, but it didn't work, so I need to be explicit.